From 95e4a5f9cbe72852bccfba90f6d3632986134d21 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 19 May 2023 17:36:10 +0200
Subject: [PATCH 0672/1695] dts: stm32: enable SAES on stm32mp257f-ev1

Sets SAES peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 00eaaa1cd72d1650b0111dd7d1655d5e288e4776 dts: stm32: enable SAES on stm32mp257f-ev1]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I0b98f1f0c62f9d3656ed184b89d52ad3e430c4d9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/361857
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 6ffc7dfe1..ad630ffba 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -418,6 +418,10 @@
 	status = "okay";
 };
 
+&saes {
+	status = "okay";
+};
+
 &scmi_regu {
 	voltd-vddcore {
 		reg = <VOLTD_SCMI_STPMIC2_BUCK2>;
-- 
2.47.2

